### PR TITLE
(DOCS-2753) Remove 'very' and 'highly' from docs

### PR DESCRIPTION
**Pull request by jtappa**
### What does this PR do?
Removes "very" and "highly" from the docs. These words are subjective and often don't actually clarify what they are "very" and "highly" whatever compared to... normal? Plus, it's just better english to avoid "very" when possible.

### Motivation

OKR!

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
(Review from Serverless Product not necessary in this case, it was added automatically due to the files in the commits)

File: content/en/database_monitoring/troubleshooting.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ Longer queries may not show their full SQL text due to database configuration. S
 {{< tabs >}}
 {{% tab "Postgres" %}}
 
-The Postgres setting `track_activity_query_size` indicates the maximum size of the SQL statement Postgres will store and make visible to the Agent. By default, this value is 1024 bytes. Raising this value to 4096 will capture most queries for most workloads. However, a higher value may be appropriate if your queries are highly complex or use very long arrays.
+The Postgres setting `track_activity_query_size` indicates the maximum size of the SQL statement Postgres will store and make visible to the Agent. By default, this value is 1024 bytes. Raising this value to 4096 will capture most queries for most workloads. However, a higher value may be appropriate if your queries are complex or use long arrays.
```
Comment (sarina-dd) [inline content/en/database_monitoring/troubleshooting.md:234]:
```suggestion
The Postgres setting `track_activity_query_size` indicates the maximum size of the SQL statement Postgres stores and makes visible to the Agent. By default, this value is 1024 bytes. Raising this value to 4096 captures most queries for most workloads. However, a higher value may be appropriate if your queries are complex or use long arrays.
```

File: content/en/integrations/faq/collect-custom-windows-performance-counters-over-wmi.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Collect Custom Windows Performance Counters over WMI
 kind: faq
 ---
 
-Datadog's WMI integration is a very versatile approach to collecting relevant metrics from your Windows environments. As long as you're able to query a WMI object for a numerical value, you'll be able to configure your dd-agent's WMI check to run the same query and collect its results as a metric (find a great guide [on how to retrieve WMI metrics][1]).
+Datadog's WMI integration is a versatile approach to collecting relevant metrics from your Windows environments. As long as you're able to query a WMI object for a numerical value, you'll be able to configure your dd-agent's WMI check to run the same query and collect its results as a metric (find a great guide [on how to retrieve WMI metrics][1]).
```
Comment (sarina-dd) [inline content/en/integrations/faq/collect-custom-windows-performance-counters-over-wmi.md:6]:
```suggestion
Datadog's WMI integration is a versatile approach to collecting relevant metrics from your Windows environments. As long as you're able to query a WMI object for a numerical value, you can configure your dd-agent's WMI check to run the same query and collect its results as a metric. See the [how to retrieve WMI metrics][1] guide.
```

File: content/en/integrations/faq/how-can-i-collect-more-metrics-from-my-sql-server-integration.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ kind: faq
...
 
-Note that, at this time, our sqlserver check only queries data from the [sys.dm_os_performance_counters][3] table (although you can [use WMI to expose metrics][4] from other counter tables). So the trick to collecting specific is to find which counter_name and (if appropriate) instance_name correspond to the kind of metric you're interested in collecting. To get a list of what count_names are available in that table in your SQL Server, once you access your server from [powershell's sqlcmd][5], you can run the following or similar query (just know that it gives you a very long list indeed):
+Note that, at this time, our sqlserver check only queries data from the [sys.dm_os_performance_counters][3] table (although you can [use WMI to expose metrics][4] from other counter tables). So the trick to collecting specific is to find which counter_name and (if appropriate) instance_name correspond to the kind of metric you're interested in collecting. To get a list of what count_names are available in that table in your SQL Server, once you access your server from [powershell's sqlcmd][5], you can run the following or similar query (just know that it gives you a long list indeed):
```
Comment (sarina-dd) [inline content/en/integrations/faq/how-can-i-collect-more-metrics-from-my-sql-server-integration.md:10]:
```suggestion
At this time, the Datadog sqlserver check only queries data from the [sys.dm_os_performance_counters][3] table, although you can [use WMI to expose metrics][4] from other counter tables. To collect specific data, find `counter_name` and, when applicable, `instance_name` to correspond to the metric you're interested in collecting. Once you access your server from [powershell's sqlcmd][5], run the following or similar query to get a list of what `count_names` are available in that table in your SQL Server. 

**Note**: This returns a long list.
```

File: content/en/logs/guide/control-sensitive-logs-data.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ Once sensitive data in logs is sent to the Datadog platform, it may exist in a n
 
 * [Log Monitors][10] notifications when they include [Log Samples][11]. Check specifically for monitors triggered during the period when sensitive data was being sent.
 
-* [Livetail][12], where logs are viewed in real-time by your organization's users. There is no persistence beyond the 50 logs cached in browsers, and for broader queries, the result can be highly sampled.
+* [Livetail][12], where logs are viewed in real-time by your organization's users. There is no persistence beyond the 50 logs cached in browsers, and for broader queries, the result can be extremely sampled.
```
Comment (sarina-dd) [inline content/en/logs/guide/control-sensitive-logs-data.md:60]:
```suggestion
* [Live Tail][12], where logs are viewed in real-time by your organization's users. There is no persistence beyond the 50 logs cached in browsers, and for broader queries, the result can be extremely sampled.
```

File: content/en/logs/log_collection/php.md [hunk 5]
```diff
@@ -236,7 +236,7 @@ Monolog comes with a pre-processor feature. It's a simple callback that enriches
 {{% /tab %}}
 {{% tab "PHP Zend-Log" %}}
 
-Much of the useful information comes from additional context data that you can add to your logs and your events. Zend-Log makes this very convenient by providing methods to set thread local context data that is then submitted automatically with all events. At any moment, log an event with contextual data:
+Much of the useful information comes from additional context data that you can add to your logs and your events. Zend-Log makes this convenient by providing methods to set thread local context data that is then submitted automatically with all events. At any moment, log an event with contextual data:
```
Comment (sarina-dd) [inline content/en/logs/log_collection/php.md:239]:
```suggestion
Useful logging information comes from additional context data that you can add to your logs and events. Zend-Log makes this convenient by providing methods to set thread local context data that is then submitted automatically with all events. At any moment, log an event with contextual data:
```

File: content/en/monitors/faq/anomaly-monitor.md [hunk 6]
```diff
@@ -13,35 +13,35 @@ No. Anomaly detection assists with visualizing and monitoring metrics that have
 
 Adding many separate timeseries in a single graph can lead to [spaghettification][1], and the problem gets worse when the anomaly detection visualization is added:
 
-{{< img src="monitors/monitor_types/anomaly/spaghetti.png" alt="spaghetti"  style="width:80%;">}}
+{{< img src="monitors/monitor_types/anomaly/spaghetti.png" alt="spaghetti" style="width:80%;">}}
```
Comment (sarina-dd) [inline content/en/monitors/faq/anomaly-monitor.md:16]:
```suggestion
{{< img src="monitors/monitor_types/anomaly/spaghetti.png" alt="Spaghettification" style="width:80%;">}}
```

**Review by sarina-dd — COMMENTED** [review]
Editorial review completed 😸

**Review by jeremy-lq — APPROVED** [review]

**Pull request merged**